src/controllers/itemDetails/index.js

```diff
@@ -1458,11 +1458,17 @@ function renderChildren(page, item) {
         imageLoader.lazyChildren(childrenItemsContainer);
         if (item.Type == 'BoxSet') {
             const collectionItemTypes = [{
-                name: globalize.translate('HeaderVideos'),
-                mediaType: 'Video'
+                name: globalize.translate('Movies'),
+                type: 'Movie'
             }, {
                 name: globalize.translate('Series'),
                 type: 'Series'
+            }, {
+                name: globalize.translate('Episodes'),
+                type: 'Episode'
+            }, {
+                name: globalize.translate('Videos'),
```

Comment:
It doesn't look like "Videos" is a valid translation key:

```suggestion
                name: globalize.translate('HeaderVideos'),
```